servers/src/main/java/tachyon/worker/block/allocator/MaxFreeAllocator.java

```diff
@@ -27,6 +27,7 @@
 
 /**
  * An allocator that allocates a block in the storage dir with most free space.
+ * The implementation prefers to get the dir from lower-level storage
```

Comment:
Maybe add `allocates a block in the storage dir with the most bytes of free space` to show its an absolute comparison not percentage. Also should clarify it prefers to use lower-level storage in the case of a tie.